jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -719,3 +720,26 @@ function configure_server_state() {
 function generate_random_id() {
     cat /dev/urandom | env LC_CTYPE=C tr -dc 'a-zA-Z0-9' | fold -w 16 | head -n 1
 }
+
+function configure_jbpm_cluster(){
+    if [ ${KIE_JBPM_CLUSTER} = 'true' ]; then
```

Comment:
use var expansion to upper case the user`s input, like this example:
https://github.com/jboss-container-images/jboss-kie-modules/pull/434/files#diff-3e7628a0c0671f1c47f2f4da279754577748d791191318fb85c3d250b1ca182cL36

Reply:
and use double quotes

Reply:
Why uppercase ?